docs/guides/building/windows.md

```diff
@@ -24,40 +24,104 @@ tauri build --target i686-pc-windows-msvc
+### Embedding the Webview2 Bootstrapper
+
+To embed the Webview2 Bootstrapper, set the [webviewInstallMode] to `embedBootstrapper`. This increases the installer size by around 1.8MB, but increases compatibility with Windows 7 systems.
+
+```json
+{
+  "tauri": {
+    "bundle": {
+      "windows": {
+        "webviewInstallMode": {
+          "type": "embedBootstrapper"
+        }
+      }
+    }
+  }
+}
+```
+
+### Embedding the Webview2 offline installer
+
+To embed the Webview2 Bootstrapper, set the [webviewInstallMode] to `offlineInstaller`. This increases the installer size by around 127MB, but allows your application to be installed even if internet connection is not available.
 
-1. Download the Webview2 fixed version runtime from the [official website][download the webview2 runtime], a `.cab` file for the selected architecture. In this example, the downloaded filename is `Microsoft.WebView2.FixedVersionRuntime.98.0.1108.50.x64.cab`
-2. Extract the file to the core folder:
+```json
+{
+  "tauri": {
+    "bundle": {
+      "windows": {
+        "webviewInstallMode": {
+          "type": "offlineInstaller"
+        }
+      }
+    }
+  }
+}
+```
 
-   ```powershell
-   expand .\Microsoft.WebView2.FixedVersionRuntime.98.0.1108.50.x64.cab -F:* ./src-tauri
-   ```
+### Using a fixed version of the Webview2 Runtime
 
-3. Configure the Webview2 runtime path on `tauri.conf.json`:
+Using the runtime provided by the system is great for security as the webview vulnerability patches are managed by Windows. If you want to control the Webview2 distribution on each of your applications, either to manage the release patches yourself or distribute applications on environments where internet connection might not be available. In that case, Tauri can bundle the runtime files for you.
 
-   ```json
-   {
-     "tauri": {
-       "bundle": {
-         "windows": {
-           "webviewFixedRuntimePath": "./Microsoft.WebView2.FixedVersionRuntime.98.0.1108.50.x64/"
-         }
-       }
-     }
-   }
-   ```
+- Download the Webview2 fixed version runtime on the [official website], a `.cab` file for the selected architecture. In this example, the downloaded filename is Microsoft.WebView2.FixedVersionRuntime.98.0.1108.50.x64.cab
+- Extract the file to the core folder: `Expand .\Microsoft.WebView2.FixedVersionRuntime.98.0.1108.50.x64.cab -F:* ./src-tauri`
+- Configure the Webview2 runtime path on `tauri.conf.json`:
 
-4. Run `tauri build` to produce the Windows Installer with the fixed Webview2 runtime.
+```json
+{
+  "tauri": {
+    "bundle": {
+      "windows": {
+        "webviewInstallMode": {
+          "type": "fixedRuntime",
+          "path": "./Microsoft.WebView2.FixedVersionRuntime.98.0.1108.50.x64/"
+        }
+      }
+    }
+  }
+}
+```
+
+- Run `tauri build` to produce the Windows Installer with the fixed Webview2 runtime.
```

Comment:
```suggestion
4. Run `tauri build` to produce the Windows Installer with the fixed Webview2 runtime.
```

Not sure if the number bullet formatting will apply correctly, can fix after committing all of these changes